x-pack/plugins/security_solution/public/explore/containers/risk_score/all/index.tsx

```diff
@@ -84,11 +84,13 @@ export const useRiskScore = <T extends RiskScoreEntity.host | RiskScoreEntity.us
   includeAlertsCount = false,
 }: UseRiskScore<T>): RiskScoreState<T> => {
   const spaceId = useSpaceId();
-  const isNewRiskScoreModuleAvailable = useIsExperimentalFeatureEnabled('riskScoringRoutesEnabled');
+  const { data: riskScoreEngineStatus } = useRiskEngineStatus();
+  const isNewRiskScoreModuleInstalled =
```

Comment:
You could use `useIsNewRiskScoreModuleInstalled` here instead of `useRiskEngineStatus`, right? You don't appear to be using the other status info.